client/components/App.js

```diff
@@ -1,17 +1,23 @@
 var React = require('react');
+var { connect } = require('react-redux');
+
 var styles = require('./App.css');
 
-var App = React.createClass({
+var App = connect(state => ({count: state}))(React.createClass({
```

Comment:
Makes sense, cheers for your work on this!

> On 29 Jan 2016, at 21:48, John R. Pelt <notifications@github.com> wrote:
> 
> In client/components/App.js:
> 
> >  var styles = require('./App.css');
> >  
> > -var App = React.createClass({
> > +var App = connect(state => ({count: state}))(React.createClass({
> Oh, state is actually just {num: 5}. if you don't name it then the prop would have this.props.num .
> 
> As for why I renamed state to count, it was because when I first started looking at the project I kept getting this.props.state confused with react's this.state. So I renamed it to make it more clear to myself.
> 
> —
> Reply to this email directly or view it on GitHub.
>